webpack.config.js

```diff
@@ -35,7 +35,7 @@ module.exports = {
     })
   ],
   optimization: {
-    minimize: true,
+    minimize: false,
```

Comment:
Could you please share your view on why you think minimization should be disabled?

Reply:
I forgot to mention this. I encountered an error once and the minimization made it very hard to debug. The difference is only 156KB, I think that is not enough to warrant difficult and likely debugging.

Reply:
Probably defaulting to `true`, but optionally checking `process.env.NODE_ENV` (which should be set to `true` in the GH actions env) would be best.